### PR TITLE
page to confirm good delivery by recipient/sender

### PR DESCRIPTION
**Pull request by tonia4ik-s**

### CODE REVIEW

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        data.updateOffers();
+    }
+
+    const confirmDelivery = async (offer, response) => {
+        let model = {
+            offerId: offer.id,
+            tripRole: offer.creatorRoleName,
+            isConfirmed: response
+        };
+        await confirmGoodsTransfer(model);
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Add the blank line

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        updateOffers()
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Add the semicolon

File: src/constants/api/urls.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export const SERVER_URL = "https://localhost:44319/api";
+export const SERVER_URL = "https://localhost:5000/api";
```
Comment (marynakernychna) [inline src/constants/api/urls.js]: Return the server url

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    }
+
+    return (
+        <Card className="offerCard">
+            <div className="cardHead">
+                <p className="creationDate">{moment(data.info.creationDate).format('LL HH:mm')}</p>
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Please, rewrite to moment.utc(data.info.creationDate).format('LL HH:mm')

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                <Text strong className="offerStatus">
+
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Remove the blank line

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+                            >
+                                I gave it away!
+                            </Button> :
+                            <Button
+                                className="submitButton"
+                                type="primary"
+                                onClick={() => confirmDelivery(data.info, true)}
+                            >
+                                I got it!
+                            </Button>}
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Move the bracket to the new line

File: src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+                    <div className="buttons-group">
+                        {data.info.creatorRoleName === offerRoles.SENDER.toUpperCase() ?
+                            <Button
+                                className="submitButton"
+                                type="primary"
+                                onClick={() => confirmDelivery(data.info, true)}
+                            >
+                                I gave it away!
+                            </Button> :
+                            <Button
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/OfferToConfirm/index.jsx]: Put a colon between the tags

File: src/components/goodsDeliveryConfirms/index.jsx [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+            {offers != null ?
+                <div className="offers-container">
+                    {offers.items.map((offer, key) =>
+                        <OfferToConfirm
+                            info={offer}
+                            updateOffers={() => updateOffers()}
+                        />
+                    )}
+                    <Pagination
```
Comment (marynakernychna) [inline src/components/goodsDeliveryConfirms/index.jsx]: Add a blank line

**Review by marynakernychna — COMMENTED** [review]
& align the code in .jsx files

**Review by marynakernychna — CHANGES_REQUESTED** [review]
Server URL

**Review by marynakernychna — APPROVED** [review]

**Review by AndriiChepeliuk — APPROVED** [review]

**Pull request merged**